apps/meteor/client/components/RoomAutoComplete/RoomAutoComplete.tsx

```diff
@@ -12,7 +13,11 @@ const generateQuery = (
 	selector: string;
 } => ({ selector: JSON.stringify({ name: term }) });
 
-type RoomAutoCompleteProps = Omit<ComponentProps<typeof AutoComplete>, 'filter'> & { scope?: 'admin' | 'regular' };
+type RoomAutoCompleteProps = Omit<ComponentProps<typeof AutoComplete>, 'filter'> & {
+	scope?: 'admin' | 'regular';
+	renderRoomIcon?: (props: { encrypted: IRoom['encrypted']; type: IRoom['t'] }) => ReactElement | null;
```

Comment:
Assume this behavior will be absorbed by the component if the design team agrees, making the property pointless.

Reply:
Yes, incase we plan to show this in other places as well, this behavior will be implemented by the component itself. Thanks Tasso!